uk.co.compucorp.civicrm.hrcore/tests/phpunit/CRM/HRCore/Service/DrupalUserServiceTest.php

```diff
@@ -69,4 +70,22 @@ protected function cleanup() {
     }
   }
 
+  private function loadRequiredData() {
```

Comment:
This method name is too generic, which usually leads to a god method, where devs will use it to load any type of data. Can you give it a more specific name?